RecoLocalMuon/DTSegment/python/DTMeantimerPatternReco2DAlgo_LinearDriftFromDB_CosmicData_cfi.py

```diff
@@ -7,16 +7,17 @@
 # The linear DriftFromDB algos is used.
```

Comment:
this would apply to nearly all of the config files here, right? We could clean this up at some point, though this would require some care as these files are being then referenced in a few other places ..